local-kubernetes/isapi/local-isapi-sample-kubernetes-manifest.yml

```diff
@@ -0,0 +1,166 @@
+apiVersion: apps/v1
```

Comment:
this file looks like a duplicate?

Reply:
Its not, have a closer look. This is the sample isapi - includes (isapi, sample resource and sample mvc)